kolibri/core/assets/src/views/core-modal/index.vue

```diff
@@ -212,7 +212,9 @@
     color: $core-text-default
     border: none
     position: absolute
-
+    &:focus
+      background-color: $core-grey-300
```

Comment:
looks good! I'm going to take this change and apply it more globally, to all `button` instances.